server/store/store.go

```diff
@@ -984,6 +998,105 @@ func (s *SQLStore) GetConnectedUsers(page, perPage int) ([]*storemodels.Connecte
 	return connectedUsers, nil
 }
 
+func (s *SQLStore) GetCountOfConnectedUsers() (int, error) {
+	query := s.getQueryBuilder().Select("count(*)").From(usersTableName).Where(sq.NotEq{"token": ""})
+	rows, err := query.Query()
+	if err != nil {
+		return 0, err
+	}
+	defer rows.Close()
+
+	var result int
+	if rows.Next() {
+		if scanErr := rows.Scan(&result); scanErr != nil {
+			return 0, scanErr
+		}
+	}
+
+	return result, nil
+}
+
+func (s *SQLStore) PrefillWhitelist() error {
+	page := 0
+	perPage := 100
+	for {
+		query := s.getQueryBuilder().Select("mmuserid").From(usersTableName).Where(sq.NotEq{"token": ""}).Offset(uint64(page * perPage)).Limit(uint64(perPage))
+		rows, err := query.Query()
+		if err != nil {
+			return err
+		}
+
+		count := 0
+		for rows.Next() {
+			count++
```

Comment:
i think we should only increase this count if we successfully add users to the table

Reply:
This count is for breaking the loop of getting connected users using pagination.